test/stdlib/ObjCEvilClassInitialization.swift

```diff
@@ -0,0 +1,25 @@
+// RUN: mkdir -p %t
```

Comment:
Nitpick: `%empty-directory(%t)` will also clear out any contents left over from the last run.

Reply:
Thanks. We have a bunch of tests with each one, and I happened to crib from one with `mkdir`. Seems better to do it with `empty-directory` though.